File: Programação Orientada a Objetos/Workspace_POO/CSharp_POO/ReadingData/Reading.cs
using System;
using System.Windows.Forms;

public  class Reading {
	public static void Main() {
		Form a = new Form();
		a.ShowDialog();
	}

	public static void UsingConsole() {
		Console.Write("What is your name? ");
		String userName = Console.ReadLine();
		Console.WriteLine("Welcome, {0}!", userName);
		//
		Console.ReadKey();
	}
}

File: Programação Orientada a Objetos/Workspace_POO/Java_POO/ReadingData/Reading.java
import java.util.Scanner;
import javax.swing.JOptionPane;

public class Reading {
	private static Scanner scanner;

	public static void main(String[] args) {
		usingJOptionPane();
	}

	public static void usingJOptionPane() {
		String name = JOptionPane.showInputDialog(null, "What is your name?");
		JOptionPane.showMessageDialog(null, "Welcome " + name + "!");
	}

	public static void usingScanner() {
		scanner = new Scanner(System.in);
		//
		System.out.print("What is your name? ");
		String userName = scanner.nextLine();
		System.out.printf("Welcome, %s!\n", userName);
	}
}